nano/templates/antiquems.tmpl

```diff
@@ -0,0 +1,42 @@
+<div class="item">
+	<div class="itemLabel">
+		Stored Capacity:
+	</div>
+	<div class="itemContent">
+		{{:helper.displayBar(data.charge, 0, 2000, data.active ? 'good' : 'average')}}
```

Comment:
The code limits charge to 1000 but this is set to 2000?

Reply:
That was just for testing to make sure the limits were working, the code side will change, not the template